command/bridge/common/params.go

```diff
@@ -35,6 +36,8 @@ const (
 	ChildTokenFlag         = "child-token"
 	JSONRPCFlag            = "json-rpc"
 	ChildChainMintableFlag = "child-chain-mintable"
+	txTimeoutFlag          = "tx-timeout"
+	txPollFreqFlag         = "tx-poll-freq"
```

Comment:
I disagree. I'd expose poll frequency as well and let somebody decides if he wants to customize it (otherwise we have the defaults and we are covered). This might be important, because if I'm not mistaken node providers (such as Alchemy) are charging per making RPC requests.

OTOH if it makes too much of an effort to add it back (sorry that you have added and then removed), let's leave it as it is for now. I don't want to block the PR because of that.